gradle/libs.versions.toml

```diff
@@ -83,6 +88,13 @@ vertx-postgres = { module = "io.vertx:vertx-pg-client", version.ref = "vertx.cli
 vertx-oracle = { module = "io.vertx:vertx-oracle-client", version.ref = "vertx.client" }
 scram = { module = "com.ongres.scram:client", version = "2.1" }
 spock = { module = "org.spockframework:spock-core", version.ref = "spock" }
+
+micronaut-aws-sdk-v1 = { module = "io.micronaut.aws:micronaut-aws-sdk-v1", version.ref = "micronaut-aws"}
```

Comment:
You shouldn't need those dependencies in this catalog: by importing the `micronaut-aws` catalog in `settings`, there is now a `mnAws` catalog, so instead of using `libs.micronaut.aws.sdk.v2` and friends, you can use:

```
implementation mnAws.micronaut.aws.sdk.v1
```

You can see the available libraries in https://s01.oss.sonatype.org/content/repositories/snapshots/io/micronaut/aws/micronaut-aws-bom/4.0.0-SNAPSHOT/micronaut-aws-bom-4.0.0-20230111.121818-23.toml

It appears that the dynamodb SDK is not part of the `micronaut-aws` catalog. Since I'm not familar with the Micronaut AWS module I can't tell if it's intentional or not, but it _seems_ that it should belong there.

Reply:
`micronaut-aws-sdk-v2` doesn't even include the actual AWS sdk

Reply:
I think so, yes.